contracts/AllowList.vy

```diff
@@ -4,24 +4,32 @@
 bouncers: public(HashMap[address, bool])
 guests: public(HashMap[address, bool])
 
+event GuestInvited:
+    guest: address
```

Comment:
should we add the bouncer that invited the guest?

Reply:
no, they should be separated.